server/test/serverTest.js

```diff
@@ -350,20 +351,31 @@ describe('Chatbot server', () => {
             expect(button.button_battery_level).to.be.null
         })
 
-        it('should update battery level column with values from new requests', async () => {
+        it('should update battery level column with values from new requests on both registry and sessions', async () => {
             let response = await chai.request(server).post('/flic_button_press').set('button-serial-number', unit1SerialNumber).set('button-battery-level', '100').send()
-
+            
             expect(response).to.have.status(200)
+            
+            let sessions = await db.getAllSessionsWithButtonId(unit1UUID)
+            let session = sessions[0]
             let button = await db.getButtonWithSerialNumber(unit1SerialNumber)
             expect(button.button_battery_level).to.equal(100)
+            expect(session.buttonBatteryLevel).to.equal(100)
+
 
             response = await chai.request(server).post('/flic_button_press').set('button-serial-number', unit1SerialNumber).set('button-battery-level', '1').send()
+            sessions = await db.getAllSessionsWithButtonId(unit1UUID)
+            session = sessions[0]
             button = await db.getButtonWithSerialNumber(unit1SerialNumber)
             expect(button.button_battery_level).to.equal(1)
+            expect(session.buttonBatteryLevel).to.equal(1)
 
             response = await chai.request(server).post('/flic_button_press').set('button-serial-number', unit1SerialNumber).set('button-battery-level', '0').send()
+            sessions = await db.getAllSessionsWithButtonId(unit1UUID)
+            session = sessions[0]
             button = await db.getButtonWithSerialNumber(unit1SerialNumber)
             expect(button.button_battery_level).to.equal(0)
+            expect(session.buttonBatteryLevel).to.equal(0)
         })
 
         it('should not change battery level column if subsequent requests do not provide button-battery-level', async () => {
```

Comment:
Thanks for adding to the tests, could you add the session part to this one and the three other battery-related tests in this file?